_lint_all_files.txt

```diff
@@ -0,0 +1 @@
+sh bin/
```

Comment:
Should we also lint/fix the top-level bash files in the repo? Although we would need a way to specify a linting directive to lint a directory non-recursively which I don't believe is possible right now.

Reply:
Yes, but I'm leaning toward putting the file list a _build.ham file so we can pick/specify the files more precisely. This just bluntly goes everywhere-ish. That's problematic when there's 3rd party code and I feel that adding yet-another custom-ish pseudo glob language for linting won't buy us anything vs just doing it in a ham file...